cola/models/dag.py

```diff
@@ -273,7 +273,12 @@ def get(self):
 
         self.reset()
         ref_args = utils.shell_split(self.params.ref)
-        cmd = self._cmd + ['-%d' % self.params.count] + ref_args
+        cmd = (
+            self._cmd
+            + ['-%d' % self.params.count]
+            + ref_args
+            + ['--date=%s' % self.context.cfg.get('cola.logdate')]
```

Comment:
sorry for not spotting this earlier (I saw it but forgot to note it).

When we call `get()` please also specify `default=prefs.DateFormat.DEFAULT` and then move the entirety of the `logdate.py` module into `cola/models/prefs.py`. That way we don't have to add a new module just for this small bit of config.

Reply:
If we don't specify `default=...` then it'll error out when the user doesn't have `cola.logdate` configured.

Reply:
I've added a function `logdate(context)' in `cola/models/prefs.py` and I'm calling the function instead of `self.context.cfg.get('cola.logdate'). Is that ok?

And one more thing, I made the change but I would prefer having the configs for logdate in separate file. Because  having them in `cola/models/prefs.py` would make this file messy. There are similar config files like `cola/hidpi.py`.